proposals/NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order.md

```diff
@@ -0,0 +1,78 @@
+ 3. If multiple SDL apps have the same `AppHMIType` set, sort them according to the order the SDL apps registered in the policy table.
+
+<b>Table 1.</b> AppHMIType priority list
+
+| Priority | AppHMIType |
+|:-:|:-:|
+| 1 | NAVIGATION |
+| 2 | MEDIA |
+| 3 | MESSAGE |
+| 4 | INFORMATION |
+| 5 | COMMUNICATION |
+| 6 | SOCIAL |
+| 7 | PROJECTION |
+| 8 | REMOTE_CONTROL |
+| 9 | SYSTEM |
+| 10 | DEFAULT |
+| 11 | BACKGROUND_PROCESS |
+| 12 | TESTING |
+
+At first, the priority information of `AppHMIType` in Table 1 is stored in the policy table, SDL Core, and default policy table (sdl_preloaded_pt.json). Then, the first searched information is used as the `AppHMIType` priority list (priority list). During the first launch, there is no policy table, therefore it is searched in the default policy table at first, then can be searched in SDL Core to obtain the default `appHMIType` priority list. After that, the policy table has been created, so it is searched in the order: policy table->default policy table->SDL Core.
+
+However, since the preferred `AppHMIType` priority may vary for each OEM, the OEMs can create the priority list information of `AppHMIType` and store it in their policy table. By changing the priority list of `AppHMIType` created by each OEM in the policy table, they can display the SDL app icon according to each OEM's intended order.
+
+<b>2. Clarification of icon display sequence</b>
+Currently, there is no sequence provided that demonstrates how the icon is displayed. Figure 1 shows the sequence in which the process of this proposal is added to the existing sequence from the program code of SDL Core. The black text indicates the existing order described in the program code. The red text indicates the process added by this proposal.
+
+<b>Figure 1.</b> SDL App Icon display sequence
+![Figure1_SDL_App_Icon_display_sequence.PNG](../assets/proposals/NNNN-Clarification-of-SDL-App-Icon-display-sequence-and-display-order/Figure1_SDL_App_Icon_display_sequence.PNG)
+
+The contents suggested by this proposal are as follows:
+- Add `SortList()` as a process to sort according to the display order of SDL app icon.
+- Perform the processing of `SortList()` after processing the normal makelist (Applications_, Applications).
+- Add `init()` as a process to initialize the App List used in `SortList()` every launch, and keep the priority list information of `AppHMIType` acquired from policy table, etc.
+
+## Potential downsides
+
+The existing sort process is left as it is. Furthermore, as process/ List related to Sort is added, SDL Core is affected. Also, since the priority information of AppHMIType is stored, policy table is affected, too. However, I believe their change are minor version change because there are no changes in RPC and parameters.
+
+## Impact on existing code
+
+This proposal has no impact on the existing code because there are no changes in RPC and parameters.
+
```

Comment:
```suggestion
## Potential downsides

There is no potential downside since it is a modification to the current process of the SDL app icon display order.

## Impact on existing code

The existing sorting process is left as it is. Furthermore, as processes/lists related to sorting are added, SDL Core is affected. Also, since the priority information of `AppHMIType` is stored, the policy server is affected, too. However, I believe their changes are minor version changes because there are no changes in RPC and parameters.

```

Reply:
Hi, @theresalech -san,
Thank you for your prompt reply.
It's my mistake. I will modify and take a commit it soon.
Thank you!